docs/imaging/sync.rst

```diff
@@ -8,7 +8,7 @@ The two-photon miniscopes make use of multiple independent simultaneous data acq
 
 For the femtonics setup (.mesc files), the internal oscilloscope function is used to record events in sync with acquisition (see `/helpers/femto_mesc.py <https://github.com/kavli-ntnu/dj-moser-imaging/blob/master/helpers/femto_mesc.py>`_).
 
-Ingest of sync data happens in the ``Sync`` table (see /dj_schemas/sync.py). For this, sync data is extracted according to dataset type (*Wavesurfer* vs. *MESC* vs. ...). Each experiment type (e.g. *femtonics*,*2Pminiscope_A*) has an entry under ``Setup``, which holds at least the *polarity* and for Wavesurfer files also info about which stream served as master and what tolerance to allow between the end of sync streams. The ``Naming`` part table allows downstream processes to infer what specific sync streams were called and to retrieve them accordingly. For example, if a table requires sync data for rotary (wheel) tracking data, it will query the ``Naming`` table to retrieve the name of the sync stream for that particular setup. That sync stream can then be retrieved from the main ``Sync`` table. 
+Ingest of sync data happens in the ``Sync`` table (see /dj_schemas/sync.py). For this, sync data is extracted according to dataset type (*Wavesurfer* vs. *MESC* vs. ...). Each experiment type (e.g. *femtonics*, *2PminiscopeA*) has an entry under ``Setup``, which holds at least the *polarity* and for Wavesurfer files also info about which stream served as master and what tolerance to allow between the end of sync streams. The ``Naming`` part table allows downstream processes to infer what specific sync streams were called and to retrieve them accordingly. For example, if a table requires sync data for rotary (wheel) tracking data, it will query the ``Naming`` table to retrieve the name of the sync stream for that particular setup. That sync stream can then be retrieved from the main ``Sync`` table. 
```

Comment:
`2Pminiscope_A` (underscore) is actually correct (old version) (see `ExperimentType()`)

Reply:
That was a botched attempt to fix formatting (rst is better than Markdown in many respects, but it doesn't support nested formatting, for some reason).

Related: I think the entire paragraph is out of date: it refers to the table `Setup.Naming`, which doesn't exist - it should probably point to `Setup.Sync` and the `generic_name` column.

Reply:
Expanding on the Sync documentation is a whole PR to itself, though, so I'm not going to worry about that right now